presto-docs/src/main/sphinx/installation/jdbc.rst

```diff
@@ -104,4 +104,7 @@ Name                              Description
 ``KerberosConfigPath``            Kerberos configuration file.
 ``KerberosKeytabPath``            Kerberos keytab file.
 ``KerberosCredentialCachePath``   Kerberos credential cache.
+``extraCredentials``              Extra credentials for connecting to external services. The
+                                  extraCredentials is a list of key-value pairs. Example:
+                                  ``foo:bar;abc:xyz`` will create credentials ``{ abc=xyz, foo=bar}``
```

Comment:
Nit: there is a space after `{` but not before `}`

How about changing this to
```
will create credentials ``abc=xyz`` and ``foo=bar``